server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -278,7 +278,7 @@ public enum APIBlock implements Writeable {
         APIBlock(String name, ClusterBlock block) {
             this.name = name;
             this.settingName = "index.blocks." + name;
-            this.setting = Setting.boolSetting(settingName, false, Property.Dynamic, Property.IndexScope);
+            this.setting = Setting.boolSetting(settingName, false, Property.Dynamic, Property.IndexScope, Property.ServerlessPublic);
```

Comment:
`read_only_allow_delete` is marked as Orchestrator in the attached G spreadsheet, while here is set as Public like all the others - am I reading it wrong?

Reply:
you are absolutely right. good catch